caim_base/models/animals.py

```diff
@@ -181,6 +181,15 @@ def ageText(self):
     def sizeText(self):
         return Animal.AnimalSize[self.size].label
 
+    def behaviourDogsText(self):
```

Comment:
so it's interesting to me that this isn't a `@property`. Does Django HTML templating not like that?

Reply:
Nice catch; I mindlessly followed the existing pattern in the model, but you're right, Django models have [a `get_foo_display` method](https://docs.djangoproject.com/en/dev/ref/models/instances/#django.db.models.Model.get_FOO_display). I'll get these text attributes replaced with that.